libraries/botbuilder/tests/teamsInfo.test.js

```diff
@@ -1150,6 +1150,869 @@ describe('TeamsInfo', function () {
+
+            const { expectedAuthHeader, expectation: fetchOauthToken } = nockOauth();
+
+            const sendMessageToListOfUsersExpectation = nock('https://smba.trafficmanager.net/amer')
+                .post('/v3/batch/conversation/users')
+                .matchHeader('Authorization', expectedAuthHeader)
+                .reply(201, { operationId: '1' });
+
+            const context = new TestContext(teamActivity);
+            context.turnState.set(context.adapter.ConnectorClientKey, connectorClient);
+
+            const operationId = await TeamsInfo.sendMessageToListOfUsers(context, activity, tenantId, members);
+
+            assert(fetchOauthToken.isDone());
+            assert(sendMessageToListOfUsersExpectation.isDone());
+            assert(operationId, { operationId: '1' });
+        });
+
+        it('should return standard error response if a 4xx status code was returned', async function () {
+            const activity = MessageFactory.text('Message to users from batch');
+            const tenantId = 'randomGUID';
+            const members = [
+                { id: '19:member-1' },
+                { id: '19:member-2' },
+                { id: '19:member-3' },
+                { id: '19:member-4' },
+                { id: '19:member-5' },
+            ];
+
+            const { expectedAuthHeader, expectation: fetchOauthToken } = nockOauth();
+
+            const errorResponse = { error: { code: 'BadSyntax', message: 'Payload is incorrect' } };
+
+            const sendMessageToListOfUsersExpectation = nock('https://smba.trafficmanager.net/amer')
+                .post('/v3/batch/conversation/users')
+                .matchHeader('Authorization', expectedAuthHeader)
+                .reply(400, errorResponse);
+
+            const context = new TestContext(teamActivity);
+            context.turnState.set(context.adapter.ConnectorClientKey, connectorClient);
+
+            let isErrorThrown = false;
+            try {
+                await TeamsInfo.sendMessageToListOfUsers(context, activity, tenantId, members);
+            } catch (e) {
+                assert.deepEqual(errorResponse, e.errors[0].body);
+                isErrorThrown = true;
+            }
+
+            assert(isErrorThrown);
+
+            assert(fetchOauthToken.isDone());
+            assert(sendMessageToListOfUsersExpectation.isDone());
+        });
+
+        it('should throw an error if an empty activity is provided', async function () {
+            const activity = null;
+            const tenantId = 'randomGUID';
+            const members = [{ id: '19:member-1' }];
+            const { expectedAuthHeader, expectation: fetchOauthToken } = nockOauth();
+
+            const sendMessageToListOfUsersExpectation = nock('https://smba.trafficmanager.net/amer')
+                .post('/v3/batch/conversation/users')
+                .matchHeader('Authorization', expectedAuthHeader)
+                .reply(201, {});

```

Comment:
Do we need nock for the tests that are asserting internal method's validations? Maybe we can just use an `assert.throws()` or `assert.rejects()`